data/adapters/topld_adapter.py

```diff
@@ -1,40 +1,74 @@
 import csv
 
 from adapters import Adapter
+from adapters.helpers import build_variant_id
 
-# Example TOPLD input file:
-# variant_1,variant_2,Uniq_ID_1,Uniq_ID_2,R2,Dprime,+/-corr
-# 2781514,2781857,2781514:C:A,2781857:C:CT,0.233,0.809,+
+# Example TOPLD input data file:
+
+# SNP1,SNP2,Uniq_ID_1,Uniq_ID_2,R2,Dprime,+/-corr
+# 5031031,5032123,5031031:C:T,5032123:G:A,0.251,0.888,+
+# 5031031,5063457,5031031:C:T,5063457:G:C,0.443,0.832,+
+
+# Example TOPLD annotation file:
+
+# Position,rsID,MAF,REF,ALT,Uniq_ID,VEP_ensembl_Gene_Name,VEP_ensembl_Consequence,CADD_phred,fathmm_XF_coding_or_noncoding,FANTOM5_enhancer_expressed_tissue_cell
+# 5031031,rs1441313282,0.010486891385767793,C,T,5031031:C:T,FP565260.3|FP565260.3|FP565260.3|FP565260.3|FP565260.3,"intron_variant|intron_variant|intron_variant|intron_variant,NMD_transcript_variant|intron_variant",2.135,.,.
 
 
 class TopLD(Adapter):
-    DATASET = 'topld'
+    DATASET = 'topld_linkage_disequilibrium'
+
+    def __init__(self, filepath_prefix=None, chr='all', ancestry='SAS'):
```

Comment:
chr can't be 'all' if this is used to build variant id?

Reply:
I will move `chr` to a regular parameter and remove the default value